file-formats/intm/examples/saplvfkd32izflkp5or2uqi1jlc4.intm.json

```diff
@@ -0,0 +1,35 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Model creation test via ADT",
+    "originalLanguage": "en"
+  },
+  "intelligentModel": {
+    "name": "RM_TEST_ADT_959_MOD",
+    "intelligentScenario": "RM_TEST_ADT_959",
+    "modellingContextId": "0405E348470277F0EBDAEF4E3CEC2C31",
+    "modelType": "SAPGENAI",
+    "adapterId": "SAPGENAI.Default",
+    "nativeModelType": "SAPGENAI.Default",
+    "state": "Published",
+    "executableId": "azure-openai",
+    "llmName": "gpt-4",
+    "llmModelVersion": "0613"
+  },
+  "promptTemplates": [
+    {
+      "name": "SYSTEM_PROMPT",
+      "description": "System prompt",
+      "prompt": "You are a sales executive who is responsible for reaching out to customers for confirming or declining their orders. Evaluate the size of the delivery vehicle based on the number of items. Calculate the distance between the billing and delivery location in kilometers. \n\nThe subject of email should be of format <Confirmation on order number: {order_number} placed on {booking_date}> \nThe body of email should be of format: \"\"\"\"SO contains <Category> to be delivered to <Location>. And will be delivered via a <size> vehicle. \nIF <distance between source address and delivery address> > 100 kms THEN include a message about potential delivery delays ELSE congratulate on being eligible for one day delivery",
+      "promptParameters": "[]",
+      "isVisible": true
+    },
+    {
+      "name": "USER_PROMPT",
+      "description": "User prompt",
+      "prompt": "Build a confirmation mail for a sales order containing {ISLM_Items} where the items are categorized for example pencil and paper will be stationary, printer and mouse will be electronics. The source address contains {ISLM_Source} and the delivery address is in {ISLM_Destination}.",
+      "promptParameters": "[{\"Name\":\"ISLM_Items\",\"Value\":\"\"},{\"Name\":\"ISLM_Source\",\"Value\":\"\"},{\"Name\":\"ISLM_Destination\",\"Value\":\"\"}]",
```

Comment:
Yes it's only Name & Value.

Reply:
I just unresolved the question. Because I want to continue on this.

If it is always a list of name and value I suggest to add this as internal table of name and value in the AFF. This would IMHO two advantages: 

1.  We get rid of the escaping due to JSON in a string field. This will make the file format much more readable
2. Users can maintain (even a table with name & values) instead of typing JSON in a String field

Reply:
Prompt parameters are dynamically derived from the prompt field with notation {ISLM_<Parameter name>}. So, we cannot declare this fields. So, we are discussing this possibility on how to provide in the UI for input. Also, prompts can be defined without parameters. So, as of now, we made it read-only and storing it as JSON string format. So, once our discussion is over, we will take this in next scope only.